include/web/app/events.py

```diff
@@ -2,10 +2,6 @@
 
 
 class App(Event):
-    WillStart = "App.WillStart"
-    """
-    Called before the app is started.
-    """
```

Comment:
Thanks

Reply:
I didn't clean-up the mess that wasn't related to the core architecture.